server/pkg/git/repo.go

```
@@ -0,0 +1,698 @@
+	if err != nil || back == "." || strings.HasPrefix(back, "..") {
+		return fmt.Errorf("invalid path %q", rel)
+	}
+	if err := os.RemoveAll(abs); err != nil {
+		return err
+	}
+	for parent := filepath.Dir(abs); parent != dir; parent = filepath.Dir(parent) {
+		if err := os.Remove(parent); err != nil {
+			break
+		}
+	}
+	return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 693) but it returns nil

(nilerr)